eva-accession-release/src/main/resources/create_hardlinks_for_unchanged_species.py

```diff
@@ -0,0 +1,74 @@
+
+import csv
+import glob
+import logging
+import subprocess
+import os
+from argparse import ArgumentParser
+from __init__ import *
+import create_assembly_name_symlinks
+
+
+species_name_mapping_file = 'species_name_mapping.tsv'
```

Comment:
Would suggest using the [release_species_inventory table](https://www.ebi.ac.uk/panda/jira/browse/EVA-2121?focusedCommentId=337750&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-337750)

Reply:
agreed offline to not use the table for now.